using-directus-as-a-baby-health-tracker/index.md

```diff
@@ -0,0 +1,403 @@
+
+If everything works so far, the JSON we receive looks something like this, but for security reasons, some data was removed or masked as it also contains coordinates, MAC addresses, IP addresses and so on:
+```json
+{
+  "status": 200,
+  "statusText": "OK",
+  "headers": {
+    ...
+  },
+  "data": [
+    {
+      "device": {
+        "dsn": "AC00***697",
+        "lan_ip": "192.168.*.*",
+        "lan_enabled": false,
+        "has_properties": true,
+        "connection_status": "Online",
+        "lat": "xxxxxxxx",
+        "lng": "xxxxxxxx",
+        "device_type": "Wifi"
+      }
+    }
+  ]
+}
+```
+
+The device object has exactly one tracker and provides us with the `dsn` as well as the flag for `has_properties`. With the next operation, we will request all of these properties to extract the sensor data from the tracker.
+
+The last Web-Request to get the properties of the device looks like this:
+
+| Method | URL |
+| --- | --- |
+| GET | `https://ads-field-eu-1a2039d9.aylanetworks.com/apiv1/dsns/{{$last.data[0].device.dsn}}/properties.json` |
+As a header parameter, we set the following:
+
+| Header | Value |
+| ---- | ---- |
+| Authorization | `auth_token {{token_sign_in.data.access_token}}` |
+As you can see, the named operation becomes handy now.
+
+This request returns every property the device currently has. In total, we received over 1600 lines of JSON this time. The good thing, we only have a single property object here. All properties have a name attribute. A very easy way is to loop over all properties and find the one we need. To achieve this within the flow, the Script-Operation can be used.
+
+The run operation looks like this
+   ```Javascript
+module.exports = async function(data) {
+    //console.log(data.read_dns.data[0])
+    let rtnData
+	data.read_dns.data.forEach(p => {
+      if (p.property.name === "REAL_TIME_VITALS") {
+        //console.log(JSON.parse(p.property.value))
+        rtnData = JSON.parse(p.property.value)
+      }
+    });
+    
+    return rtnData
+}
+```
+
+Within `rtnData` we now store all the relevant values such as battery details, heart rate and oxygen level. As the JSON data is stored as a raw string with the value attribute, the run operation already parses the string and returns the following JSON object back to the Flow
+
+```Json
+{
+  "ox": 0,
+  "hr": 0,
+  "mv": 0,
+  "sc": 2,
+  "st": 0,
+  "bso": 0,
+  "bat": 100,
+  "btt": 0,
+  "chg": 2,
+  "aps": 0,
+  "alrt": 0,
+  "ota": 0,
+  "srf": 1,
+  "rsi": 18,
+  "sb": 0,
+  "ss": 0,
+  "mvb": 0,
+  "mst": 0,
+  "oxta": 255,
+  "onm": 0,
+  "bsb": 4,
+  "mrs": 0,
+  "hw": "obl"
+}
+```
+
+Some of the keys are self-speaking while others are not clear. Due to the lack of documentation, I can only compare the values with the data from the original app to identify what is what. Also, the mentioned GitHub projects have some details from other users that were helping to understand the keys here.
```

Comment:
Give some examples, too! What stats are interesting? What is ox, hr, st, and bat? It's clear in the next step, but spell it out explicitly here.